app/jobs/calendar_importer/events/resident_advisor.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module CalendarImporter::Events
+  class ResidentAdvisor < Base
+    def initialize(event)
+      super
+      @event = event
+    end
+
+    def uid
+      @event['id']
+    end
+
+    def summary
+      @event['title']
+    end
+
+    def description
+      @event['content'] || ''
+    end
+
+    def place
```

Comment:
Should we remove altogether if N/A ? Other importers only seem to have the ones that are relevant for their events.

Reply:
yeah im just unsure if Event.place is still a thing - i.e. a ref to a partner that the Event takes place at